client/state/data-layer/wpcom/sites/activity/test/from-api.js

```diff
@@ -71,22 +65,43 @@ const API_RESPONSE_BODY = deepFreeze( {
 } );
 
 describe( 'fromApi', () => {
-	it( 'should process a valid API response', () => {
-		expect( fromApi( API_RESPONSE_BODY ) ).to.be.an( 'array' ).that.is.not.empty;
-	} );
+	context( '#schema', () => {
```

Comment:
I wasn't sure what to do with these tests. I'd like to know my schema is working as expected, but perhaps a new test file is in order that uses `is-my-json-valid` to test.

Reply:
just ask yourself what the purpose of the test is, what it's value is

if you are testing that your schema works, how will you test that? will you mock data for the validator? if so, how do we know that actually corresponds to the real API responses? it wouldn't only make sense that we adjust our mock data until the schema passes tests, which is kind of working backwards.

maybe an appropriate test could therefore be something like making sure that the schema actually compiles and nothing else. if it were me, I wouldn't spend too much time fretting it. a problem with the schema will become visible quite rapidly